README.md

```diff
@@ -4,6 +4,10 @@
 
 A low level Postgres client library for Haskell. Used at Hasura in various production projects.
 
+This library is no longer developed here, but remains in case it is used as a
+project dependency. All further updates can be found in the
+[graphql-engine](https://github.com/hasura/graphql-engine/tree/main/server/lib/pg-client-hs) repository.
```

Comment:
This wording could be misinterpreted to mean that the code isn't being developed at all anymore.
```suggestion
The code for this library has now moved to another git repository, and can be
found here: 
[graphql-engine](https://github.com/hasura/graphql-engine/tree/main/server/lib/pg-client-hs) repository.
```

We should also mark this git repository as archived.